### DISCUSSION TITLE
branch를 새로 만들어야 할까요.

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#55] [General]
브랜치는 대량의 코드를 복사해 오고 작업을 해서 PR을 거쳐 다시 merge하는 것인데 
지금은 처음부터 만드는것이니 기존 스터디룸에 해놔도 상관없겠죠?

**Comment by Woon-2** [comment]
>온라인 작업 공간이기 때문에
>만드는 것도 0 코스트, 삭제하는 것도 0 코스트라는 강력한 장점이 있습니다.
>
>무한정으로 만들었다가, 순식간에 모두 없애버릴 수도 있죠.   
>- [branch](https://github.com/Woon-2/cpp-study/blob/assignments-CS-1-CS-114/docs/branch.md)

`branch`를 얼마든지 만드셔도 됩니다.   
작업 효율을 생각해 많은 수의 `branch`를 두는 건 권장하지 않지만,    
아무런 리스크가 없습니다.   

대량의 코드를 복사하지도 않습니다.   
`git`이 `snapshot`을 기록한다는 게 어떤 의미인지 알아보시면,    
새로운 `branch`를 생성하는 게 얼마나 가벼운 동작인지 아실 겁니다.   

원하는대로 선택하세요.
